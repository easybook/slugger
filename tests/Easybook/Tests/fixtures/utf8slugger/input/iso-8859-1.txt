!"#$%&'()*+,-./
0123456789:;<=>?
@ABCDEFGHIJKLMNO
PQRSTUVWXYZ[\]^_
`abcdefghijklmno
pqrstuvwxyz{|}~
¡¢£¤¥¦§¨©ª«¬®¯
°±²³´µ¶·¸¹º»¼½¾¿
ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏ
ÐÑÒÓÔÕÖ×ØÙÚÛÜÝÞß
àáâãäåæçèéêëìíîï
ðñòóôõö÷øùúûüýþÿ