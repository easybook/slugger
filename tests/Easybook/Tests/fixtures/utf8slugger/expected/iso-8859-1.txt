
0123456789
abcdefghijklmno
pqrstuvwxyz
abcdefghijklmno
pqrstuvwxyz
cr
1-4-1-2-3-4
aaaaaaaeceeeeiiii
dnoooooouuuuythss
aaaaaaaeceeeeiiii
dnooooo-ouuuuythy